bin/distmap

```diff
@@ -457,7 +457,7 @@ sub check_inputs {
 	$args_dict->{"exec_arch"} = "DistMap_Mapper_Archive.tgz";
 	$args_dict->{"extracted_execarch"} = "execarch";
 
-	$args_dict->{"ref_arch"} = "Genome_Index_Archive.tgz";
+	$args_dict->{"ref_arch"} = "Distmap_Genome_Index_Archive.tgz";
```

Comment:
Should be a constant somewhere (like in `ReadToolsUtils.pm` the minimum required version.

Reply:
Like that, it can be shared accross packages and it's less error prone...